packages/components/src/GuidedTour/GuidedTour.scss

```
@@ -6,6 +6,9 @@ $tc-guidedtour-nav-button-size: $svg-md-size !default;
 $tc-guidedtour-controls-color: $dove-gray !default;
 
 :global(#___reactour) {
+	> div {
+		z-index: $zindex-modal;
```

if you want to be under notifications, I would suggest using:
```suggestion
		z-index: $zindex-notification - 1;
```

---

$zindex-modal < $zindex-notication. I thunk that use the same z index that the modal (the guided tour is a modal) has more sense that just do notification -1